hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/TraceabilitySuite.java

```diff
@@ -4588,7 +4580,7 @@ final HapiSpec traceabilityE2EScenario21() {
                                                 .setCallingAccount(TxnUtils.asId(GENESIS, spec))
                                                 .setCallOperationType(CallOperationType.OP_CALL)
                                                 .setGas(978936)
-                                                .setGasUsed(963748)
+                                                .setGasUsed(3172)
```

Comment:
Does this fail mono-service and work for mod service ? Why are we changing these numbers?

Reply:
Hey! We are changing the flow and returning different statuses to the user. I tough that was the reason for the change in gas. Am I wrong ?

Reply:
@Neeharika-Sompalli

Reply:
I am not sure. But changing `HapiSpec` seems like changing the logic from mono-service. 
 @tinker-michaelj  thoughts?

Reply:
@lukelee-sl , @david-bakin-sl
Could you please review this ?

Reply:
IMO, we should discourage (but still allow) this use case where the user tries to call a function on a system account.  One way to do that is if we charge all the remaining gas in the transaction but still allow the transaction to succeed.  I believe if we did this then the `gasUsed` value above in the test may not have to be updated (I haven't tested this).

Reply:
If we do this `system-accounts-operations.md` should be updated - it doesn't discuss fees for "no-op success" at all.

Reply:
@Neeharika-Sompalli @tinker-michaelj @lukelee-sl @david-bakin-sl 
We are changing the logic from mono, indeed. This fix and others from the same package are bugs found in mono codebase and has to be fixed in modu

Reply:
New version available guys :) 
In it we consume all the gas. 
https://github.com/hashgraph/hedera-services/pull/12833/files#diff-79d792e4de7dd21f5fe7e60edebaea811a3bfd113a27838a84208e9747c00cbbR174